File: swea/D2/1926.간단한 369게임/1926.간단한 369게임.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){
  int n;
  cin >> n;

  for(int i = 1; i <= n; i++){
    string s = to_string(i);
    int cnt = 0;

    for(char c : s){
      if(c == '3' || c == '6' || c == '9')
        cnt ++;
    }
    
    if(cnt > 0)
      cout << string(cnt, '-') << ' ';
    
    else
      cout << s << ' ';
  }
}

File: swea/D2/1926.간단한 369게임/1926.간단한 369게임.py
n = int(input())

for i in range(1, n + 1):
    s = str(i)
    cnt = 0

    for ch in s:
        if ch in '369':
            cnt += 1

    if cnt > 0:
        print('-' * cnt, end=' ')
    else:
        print(i, end=' ')